AC-2646 Update docs with OpenSearch (#1748)
magento/merchdocs: src/catalog/search-configuration.md

   Restricting this amount increases performance of searches and reduces the displayed list size. The default value is `8`.

### Step 2: Configure the Elasticsearch/OpenSearch connection

{:.bs-callout-info}
The **Search Engine**, **Elasticsearch Server Hostname**, **Elasticsearch Server Port**, **Elasticsearch Index Prefix**, **Enable Elasticsearch HTTP Auth**, and **Elasticsearch Server Timeout** fields were configured when Commerce was installed or upgraded. These values should only be changed when upgrading or modifying Elasticsearch.

{:.bs-callout-info}

> For Magento 2.4.6 and higher you can set up OpenSearch instead of Elasticsearch. You need to choose the value `OpenSearch` for **Search Engine** setting.

Reviewer: ```suggestion
For Adobe Commerce and Magento Open Source 2.4.6 and higher, you can set up OpenSearch instead of Elasticsearch. To use OpenSearch, choose the `OpenSearch` value for the **Search Engine** setting.
```

Author: done

![]({% link catalog/assets/elasticsearch-2.png %}){: .zoom}
[_Elasticsearch Connection Settings_]({% link configuration/catalog/catalog.md %})

1. For **Search Engine**, accept the value `Elasticsearch 7`.

>    See [here][3]{:target="_blank"} what version of the Elasticsearch is required for all Commerce installations.

Reviewer: ```suggestion
   See the [system requirements][3]{:target="_blank"} to determine the version of the Elasticsearch that is required for Commerce installations.
```

Author: done
